.github/workflows/inductor.yml

```diff
@@ -3,6 +3,7 @@ name: inductor
 on:
   schedule:
     - cron: 45 1,5,9,13,17,21 * * *
+    - cron: 29 8 * * *  # about 1:29am PDT
```

Comment:
I think we should leave inductor alone in this PR because there are lot of waste running it this way.  Across all inductor shards, only `inductor` and may be `inductor_distributed` has the benefit of mem_leak_check run.  The rest (`inductor_huggingface`, `inductor_timm`, `inductor_torchbench`) are not unit tests. So, `mem_leak_check` or not, they are running the same thing.

We could split the test matrix into 2 parts, one that supports `mem_leak_check`, and the other doesn't.  Then use a if statement to check which one to run depending on the schedule.  But this sounds better to be done in a separate PR.

Reply:
got rid of inductor for now